### PR TITLE
fix: change description that subscriptionExpireTime is optional #120

### LINKED ISSUE #161: subscriptionExpireTime is declared as "required" in the subscription-ends - note, but is optional currently
**Problem description**
[](https://github.com/camaraproject/DeviceStatus/blob/main/code/API_definitions/device-status.yaml#L82-L84)

In the note for a subscription-ends event the `subscriptionExpireTime` is declared as "required by the requester".
But this request-parameter in POST /subscriptions is not mandatory. 

**Expected behavior**
Remove "required by the requester".

### PR DESCRIPTION
**Pull request by maxl2287**
#### What type of PR is this?

Add one of the following kinds:
* correction

#### What this PR does / why we need it:

Correct the description for "subscription-ends" that this event will be send, when subscription expire time is optionally set by the requester.

#### Which issue(s) this PR fixes:

Fixes #161

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
Good for me!

File: code/API_definitions/geofencing.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ info:
     
     Note: Additionally to these list, ``org.camaraproject.geofencing.v0.subscription-ends`` notification `type` is sent when the subscription ends. 
     This notification did not requires dedicated subscription.
```
Comment (alpaycetin74) [inline code/API_definitions/geofencing.yaml:45]: Minor typo here. Should read: "This notification **does not require** dedicated subscription." 
Thank you.

**Review by alpaycetin74 — APPROVED** [review]
LGTM, thank you

**Review by bigludo7 — APPROVED** [review]

**Review by jlurien — APPROVED** [review]
This explanation is more clear now, thanks

**Review by jlurien — CHANGES_REQUESTED** [review]
Sorry, I didn't realise before that we have to update version to 0.2.0-wip

**Review by alpaycetin74 — APPROVED** [review]
LGTM, thank you

**Comment by maxl2287** [comment]
This changes will be included in #163 .
That's why this PR will be closed.

**Pull request closed without merging**